src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -80,6 +79,19 @@ class ApiOperations extends DataOperations with MetadataOperations {
     optionalMax.map(_.getOrElse(-1))
   }
 
+  /**
+    * Fetches the level of the most recent votes stored in the database.
+    *
+    * @return Max level or -1 if no votes were found in the database.
+    */
+  def fetchVotesMaxLevel()(implicit ec: ExecutionContext): Future[Int] = {
+    val optionalMax: Future[Option[Int]] = runQuery(Tables.Votes.map(_.level).max.result)
```

Comment:
just wondering... is it possible to `max.getOrElse(-1).result` directly in the query?

Reply:
By the way I guess we should have some code constant to describe `-1` as a value instead of a "magic number"

Reply:
good point, changed